implementations/elixir/ockam/ockam/lib/ockam/worker/authorization.ex

```diff
@@ -119,7 +119,7 @@ defmodule Ockam.Worker.Authorization do
   to be handled by the worker.
   """
   def from_secure_channel(prev \\ :ok, message, _state) do
-    Logger.debug("check is secure #{inspect(message)}")
+    Logger.debug("check from secure channel #{inspect(message)}")
```

Comment:
I realize this is no worse than the existing impl, but can we be any more conservative about limiting the inspect output? Do messages have a useful Inspect protocol definition?

Reply:
some explicit textual representation would be nice yes.  
So far we are logging the message this way in lots of places,  it's still "not-to-bad"  cause the things most likely to be of interest  (from/to  route,  and metadata)  are short enough and have okish text representation already.
Message' body is truncated